Signal/src/Models/OWSMessagesBubblesSizeCalculator.m

```diff
@@ -75,28 +79,40 @@ - (CGSize)messageBubbleSizeForMessageData:(id<JSQMessageData>)messageData
                               atIndexPath:(NSIndexPath *)indexPath
                                withLayout:(JSQMessagesCollectionViewFlowLayout *)layout
 {
+    id cacheKey = [self cacheKeyForMessageData:messageData];
```

Comment:
DRY.  The base class will do some redundant work updating the same cache for incoming and outgoing messages, but this is harmless and cheap.

Reply:
Nice.